plugins/paper/src/main/java/com/playmonumenta/plugins/protocollib/EntityEquipmentReplacer.java

```diff
@@ -33,14 +35,23 @@ public EntityEquipmentReplacer(Plugin plugin) {
 	@Override
 	public void onPacketSending(PacketEvent event) {
 
-		// doc: https://wiki.vg/Protocol#Entity_Equipment
+		// doc: https://wiki.vg/Protocol#Set_Equipment
 
 		PacketContainer packet = event.getPacket();
 		Entity entity = packet.getEntityModifier(event).read(0);
 		VanityManager.VanityData vanityData = entity instanceof Player player && mPlugin.mVanityManager.getData(event.getPlayer()).mOtherVanityEnabled ? mPlugin.mVanityManager.getData(player) : null;
+
 		List<Pair<EnumWrappers.ItemSlot, ItemStack>> items = packet.getSlotStackPairLists().read(0);
 		for (Pair<EnumWrappers.ItemSlot, ItemStack> pair : items) {
-			if (vanityData != null && pair.getSecond() != null && pair.getSecond().getType() != Material.AIR) {
+			// we replace the item if the player that the packet is sent to has gear disabled...
+			if(ScoreboardUtils.getScoreboardValue(event.getPlayer(), "ShouldDisplayOtherPlayerGear").orElse(1) == 0 &&
+				// and if the entity whose gear we are modifying is a player
+				entity instanceof Player &&
+				// and it is not the current player, since they need to see their own gear
+				!entity.getUniqueId().equals(event.getPlayer().getUniqueId())) {
+				pair.setSecond(new ItemStack(Material.AIR));
+			}
+			else if (vanityData != null && pair.getSecond() != null && pair.getSecond().getType() != Material.AIR) {
```

Comment:
This looks like a great addition! Thanks. 

Can you move the test logic out of the loop itself? There's no reason to check for every item slot if data should be sent. 

Also - can you swap the ordering of the conditionals in the packet listener so the scoreboard lookup is last - that way we only do score checks after the other conditions are true, rather than doing them for non-players also.

Reply:
Should be fixed with [f68434b](https://github.com/TeamMonumenta/monumenta-plugins-public/pull/9/commits/f68434b1a64e6291080bd810e021143169060a83)